File: x11/3ddesktop/files/patch-config.cpp
--- config.cpp.orig	2005-06-20 11:20:44 UTC
+++ config.cpp
@@ -64,8 +64,9 @@ get_boolean (char *value) 
 }
 
 
-Options::Options(char *n = NULL) 
+Options::Options(char *n) 
 {
+    n = NULL;
 
     if (n)
         strncpy(name, n, sizeof(name));

File: x11-toolkits/py-wxPython30/files/patch-config.py
--- config.py.orig	2015-04-11 19:08:36 UTC
+++ config.py
@@ -1030,6 +1030,7 @@ elif os.name == 'posix' or COMPILER == '
 
     cflags = getWxConfigValue('--cxxflags')
     cflags = cflags.split()
+    cflags.append('-std=c++11')
     if debug:
         cflags.append('-ggdb')
         cflags.append('-O0')
